src/frontend/screens/Settings/ProjectSettings/YourTeam/SelectDevice.tsx

```diff
@@ -31,8 +32,16 @@ export const SelectDevice: NativeNavigationComponent<'SelectDevice'> = ({
 }) => {
   const ssid = useLocalDiscoveryState(state => state.ssid);
   const {formatMessage: t} = useIntl();
-
+  const projectMembers = useProjectMembers();
   const devices = useLocalPeers();
+  const nonMemberDevices = !projectMembers.data
+    ? devices
+    : devices.filter(
+        device =>
+          !projectMembers.data.some(
+            member => member.deviceId === device.deviceId,
+          ),
+      );
```

Comment:
nit but usually a little easier to follow when using the non-negated form in cases like this e.g. `projectMembers.data ? ... : ...`

Reply:
https://github.com/digidem/comapeo-mobile/pull/206/commits/a9e79e44b113478dc5f6bc3596bdc77bb6882af9